src/client/api/init.c

```diff
@@ -53,7 +53,6 @@ const struct daos_task_api dc_funcs[] = {
 	/** Management */
 	{dc_mgmt_svc_rip, sizeof(daos_svc_rip_t)},
 	{dc_pool_extend, sizeof(daos_pool_extend_t)},
-	{dc_pool_evict, sizeof(daos_pool_evict_t)},
```

Comment:
I made some changes in this area after discussion with the team in https://github.com/daos-stack/daos/pull/3984, you may want to take a look, I think it applies here as weel.

Reply:
Thank you for sharing @GitHuaKuang, working on this.